packages/zoe/tools/fakePriceAuthority.js

```diff
@@ -79,7 +82,7 @@ export async function makeFakePriceAuthority(options) {
   const quoteIssuer = E(quoteMint).getIssuer();
   const quoteMath = await makeLocalAmountMath(quoteIssuer);
 
-  /** @type {NotifierRecord<PriceQuote>} */
+  /** @type {NotifierRecord<undefined>} */
   const { notifier, updater } = makeNotifierKit();
```

Comment:
Hmm, I am confused about why we have a notifier and an updater internally. It seems like we could take it out entirely and just have the `wake` function call `priceInQuote(amountIn, brandOut, timestamp);`. Let me know if there's something I'm missing.

Reply:
I can explain the bizarre use of an internal notifier: the timer API doesn't give a way to remove wakers from a repeater.  Only add one, or disable all.  Because of this, I need to expose the time source to other consumers.  I will call the notifier "ticker" to make this clearer and explain why we don't do the work inside the wake function.

Reply:
The repeater is only used once. Couldn't `deleteRepeater()` be used to deschedule the repeater? Does it need to schedule multiple events on a single repeater?

Reply:
Gotcha, if we add a TODO with a link to a Github issue for adding a notifier to the timerService, I'm ok with this.